tutorials/README.md

```diff
@@ -1,9 +1,22 @@
 ## Tutorials
-This folder contains tutorial notebooks for DIANNA.
-A tutorial is avaiable for several combinations of data modality and explanability method.
-Click one of the links in the table to directly go to a notebook:
+This folder contains DIANNA tutorial notebooks.
 
+Tutorials are available for several combinations of data modality, dataset and explanability method.
+
+Use the links in the tables below to directly go to a tutorial notebook:
+
+<!--- to do: check if LIME is for LeafSNap (Simple Sicnetific) or more. DO we need Images row or each notebook is for 1 dataset only!? -->
```

Comment:
The original idea was one image dataset for each method, but there are some extra notebooks. I would only do one row for images, but multiple is also fine. LIME was only done for leafsnap

Reply:
I will probably redo it. Have to see how to make this PR draft again.